test/commands/profiles/create.test.js

```diff
@@ -11,9 +11,15 @@ const helpMessages = require('../../../src/services/messaging/help-messages');
 describe('commands', () => {
   describe('profiles', () => {
     describe('create', () => {
-      const createTest = (commandArgs = [], profileId = 'default') =>
+      const createTest = (commandArgs = [], { profileId = 'default', addProjects = 0, removeCred = true } = {}) =>
         test
           .twilioFakeProfile(ConfigData)
```

Comment:
We are using  these methods in testcases`twilioFakeProfile`, `twilioCliEnv` and `twilioCreateCommand`, but I am not able to find the definitions. Are these taken care by Mocha?

Reply:
https://code.hq.twilio.com/twilio/cli-test/blob/master/src/index.js

Reply:
Thanks for pointing the repo, It has `twilioCliEnv` and `twilioCreateCommand` methods but not `twilioFakeProfile`(It used to be `twilioFakeProject` - as part Switch naming from 'project(s)' to 'profile(s)' we changed it to `twilioFakeProfile`). Even though the method is not there in cli-test our test are passing(I am not sure why).